packages/grafana-ui/src/components/MatchersUI/FieldNameMatcherEditor.tsx

```diff
@@ -32,19 +32,22 @@ export const fieldNameMatcherItem: FieldMatcherUIRegistryItem<string> = {
   optionsToLabel: (options) => options,
 };
 
-const useFieldDisplayNames = (data: DataFrame[]): Set<string> => {
+export function useFieldDisplayNames(data: DataFrame[]): Set<string> {
```

Comment:
nit: what do you think about moving this to its own file instead of just exporting it?

Reply:
no strong preference.... I just moved the logic to a util function in `@grafana/data` but I'm not totally convinced we need/want to actually add this to the API surface

Reply:
Agree, I would have probably just created a file in the MatchersUI folder that holds the function.